client/js/_author/components/assessments/question_types/audio_upload.jsx

```diff
@@ -33,15 +52,24 @@ export default class AudioUpload extends React.Component {
     });
   }
 
-  static rangeWarning(){
-    return (
-      <span className="author--c-inline-error">Please enter a number under 240</span>
-    );
+  handleTimeLimitUpdate(e) {
+    let timeLimit = e.target.value;
+    const timeVal = parseInt(timeLimit, 10);
+    let displayWarning = false;
+    if (timeVal > AudioUpload.MAX_TIME) {
+      timeLimit = `${AudioUpload.MAX_TIME}`;
```

Comment:
why do we always ``${AudioUpload.MAX_TIME}``, is there an explicit reason we need this value as a string? It may be better to leave it as an int and if you have to stringify it, toString() or _.toString() it only at the spot it needs to be a string